public/js/events.js

```
@@ -0,0 +1,243 @@
+    // Custom short code validation
+    const customShortCode = document.getElementById('customShortCode');
+    const shortCodeCounter = document.getElementById('shortCodeCounter');
+    const customShortCodeError = document.getElementById('customShortCodeError');
+    const customShortCodeSuccess = document.getElementById('customShortCodeSuccess');
+    if (customShortCode) {
+        customShortCode.addEventListener('input', () => {
+            const value = customShortCode.value;
+            if (shortCodeCounter) shortCodeCounter.textContent = value.length;
+            customShortCode.value = value.replace(/[^a-zA-Z0-9-_]/g, '');
+            if (value.length > 0) {
+                if (typeof validateCustomShortCode === 'function') validateCustomShortCode(value);
```

ESLint

[error] 'validateCustomShortCode' is not defined.

(no-undef)